packages/protocol/test/bridge/Bridge.test.ts

```diff
@@ -469,6 +465,25 @@ describe("integration:Bridge", function () {
             .deploy();
         await l2AddressManager.init();
 
+        const l1SignalService = await deploySignalService(
```

Comment:
```suggestion
        const { signalService: l1SignalService } = await deploySignalService(
```

Reply:
prevents l1SignalService from being recognized by linter as type { signalService: SignalService }

Reply:
The current approach is better as `deploySignalService` only returns one value

Reply:
My vscode shows a bunch of errors without this, and it matches the deployBridge(), maybe better for consistency? Though it does run fine without it

Reply:
<img width="607" alt="ex" src="https://user-images.githubusercontent.com/50648015/211598869-d3eed078-a683-42ac-a674-159366d45a96.png">
Specifically this issue @dantaik , I'm not sure if it's just local to my vscode